app/views/va_facilities/_index_table_row.html.erb

```diff
@@ -3,7 +3,11 @@
     <th scope="row" role="rowheader" class="grid-col-4"><a class="usa-link" href='<%= va_facility_path(facility) %>'><%= facility["official_station_name"] %> (<%= facility["common_name"] %>)</a></th>
     <td class="grid-col-1"><%= facility.street_address_state %></td>
     <td class="grid-col-1"><a class="usa-link" href='/visns/<%= facility.visn.number %>'><%= facility.visn.number %></a></td>
-    <td class="grid-col-2"><%= facility.fy17_parent_station_complexity_level[0..1].upcase %></td>
+    <% if facility.fy17_parent_station_complexity_level.match(/^1/) %>
+      <td class="grid-col-2"><%= facility.fy17_parent_station_complexity_level[0..1].upcase %></td>
+    <% else %>
+      <td class="grid-col-2"><%= facility.fy17_parent_station_complexity_level[0..0].upcase %></td>
+    <% end %>
```

Comment:
Can we do a ternary statement here? Something like: 

```
<% facility_complexity = facility.fy17_parent_station_complexity_level %>
<td class="grid-col-2"><%= facility_complexity.match(/^1/) ? facility_complexity[0..1].upcase : facility_complexity[0..0].upcase %></td>
```

Reply:
Good call Josh! I fixed it up.